src/test/java/org/assertj/core/internal/paths/Paths_assertIsAbsolute_Test.java

```diff
@@ -0,0 +1,65 @@
+package org.assertj.core.internal.paths;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.internal.PathsBaseTest_NoFileSystem;
+import org.junit.Test;
+
+import java.nio.file.Path;
+
+import static org.assertj.core.error.ShouldBeAbsolutePath.shouldBeAbsolutePath;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.test.TestFailures.failBecauseExpectedAssertionErrorWasNotThrown;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("AutoBoxing")
+public class Paths_assertIsAbsolute_Test
+    extends PathsBaseTest_NoFileSystem
```

Comment:
I would just get rid of `PathsBaseTest_NoFileSystem`, override `setup` calling `PathsBaseTest#setup` and initialising mock. Having a class `PathsBaseTest_NoFileSystem` with a `initPath` feels wrong to me.
IMO, if you need a base class for test that uses a mock path, its name should reflect that, maybe something like `WithMockPathsBaseTest` ...